core/src/com/group66/game/cannon/Ball.java

```diff
@@ -64,9 +63,6 @@
     /** The color of the Ball. */
     private BallType type;
 
-    /** The radius of the Ball. */
-    private float radius;
```

Comment:
Why remove radius? We do not want to possibly have different sizes in the future?